### ISSUE TITLE
When adding a fow, maybe give a default mask and value

### ISSUE THREAD

**Issue by muxi** [flowgrammable/flowsim#142]
For example filling the value of a field match for Ethernet src is confusing if user does not know what the format should be. A default value suggests the format.

**Comment by jassoncasey** [comment]
You are right we currently don't do a good job showing what acceptable input for input boxes in general. 

Default is usually /32 ... maybe we could show that in the match set display as well.
